python/ql/test/experimental/meta/ConceptsTest.qll

```diff
@@ -32,3 +32,39 @@ class SystemCommandExecutionTest extends InlineExpectationsTest {
     )
   }
 }
+
+class HttpServerRouteSetupTest extends InlineExpectationsTest {
+  HttpServerRouteSetupTest() { this = "HttpServerRouteSetupTest" }
+
+  override string getARelevantTag() { result in ["routeSetup", "routeHandler", "routedParameter"] }
```

Comment:
Is it more convenient somehow to have all three implementations mixed together like this? I don't see that they're sharing any actual code.

Reply:
Fair point. I think I tried making them share code initially, and just ended up leaving them in the same `InlineExpectationsTest` class. As we get more and more concepts, we will get more and more `InlineExpectationsTest` in this file as well, so I think it could help creating an overview by grouping the related tests together.

I think it gives a bit less boilerplate code, which seems nice.

All in all, not something I'm going to fight for/against, so if you feel this needs to be changed, I would be happy to.

Reply:
Ah, I see your point in having them all under the same heading. Makes sense.